.github/workflows/sync_discourse.yaml

```diff
@@ -0,0 +1,29 @@
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
+
+jobs:
+
+  sync-docs:
+    name: Open PR with docs changes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Open PR with docs changes
+        uses: canonical/upload-charm-docs@main
+        id: docs-pr
+        with:
+          discourse_host: discourse.charmhub.io
+          discourse_api_username: ${{ secrets.DISCOURSE_API_USERNAME }}
+          discourse_api_key: ${{ secrets.DISCOURSE_API_KEY }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          dry_run: "true"
```

Comment:
No, it does not actually. I believe it works with "true" (I actually tested it on the repository I showed yesterday demo), since it is actually converted in the code, [here](https://github.com/canonical/upload-charm-docs/blob/main/main.py#L42)